src/app/appfactory.cpp

```
@@ -89,24 +90,25 @@
 using namespace muse;
 using namespace au::app;
 
-std::shared_ptr<muse::IApplication> AppFactory::newApp(const CommandLineParser& parser) const
+std::shared_ptr<muse::IApplication> AppFactory::newApp(const std::shared_ptr<AudacityCmdOptions>& options) const
```

Cppcheck (2.20.0)

[style] The function 'newApp' is never used.

(unusedFunction)